pkg/network/http/http_statkeeper_windows.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build (windows && npm) || linux_bpf
+// +build windows,npm linux_bpf
+
+package http
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+)
+
+func getPathBufferSize(c *config.Config) int {
+	return int(c.HTTPMaxRequestFragment)
```

Comment:
I see why you have added this. But I wonder if existing global config alone is sufficient

Reply:
again it's runtime configurable.